hazelcast/proxy/map.py

```diff
@@ -187,7 +187,7 @@ def add_entry_listener(
             LOADED=loaded_func,
         )
 
-        if key is not None and predicate:
+        if key is not None and predicate is not None:
```

Comment:
Let's do the same in replicated_map too.

Reply:
Changed at https://github.com/hazelcast/hazelcast-python-client/pull/550/commits/0243e0c4c9c8622a2cd46c526cbe4d705638efd6